runners/spark/build.gradle

```diff
@@ -84,6 +84,12 @@ dependencies {
   compile library.java.slf4j_api
   compile library.java.joda_time
   compile library.java.args4j
+  compile project(path: ":model:fn-execution", configuration: "shadow")
+  compile project(path: ":model:job-management:", configuration: "shadow")
+  compile project(":sdks:java:fn-execution")
+  compile library.java.vendored_grpc_1_26_0
+  compile library.java.vendored_guava_26_0_jre
+  compile "org.checkerframework:checker-qual:2.0.0"
```

Comment:
This dependency was suggested to be added in BeamModulePlugin. Or at least it should have a shared definition there. It is a universal Java dependency.

Reply:
@kennknowles ,  "org.checkerframework:checker-qual" has a universal definition in https://github.com/apache/beam/pull/13690/files#diff-0435a83a413ec063bf7e682cadcd56776cd18fc878f197cc99a65fc231ef2047

Reply:
I see. So it should not be needed here. Perhaps we should very soon merge just those changers to the universal definitions and then update the smaller PRs.

Reply:
@kennknowles sure, https://github.com/apache/beam/pull/13690/ PR is ready for review.

Reply:
#13690 is merged, can you update this now?

Reply:
> #13690 is merged, can you update this now?

removed from spark build.gradle as it is reference there in Universal deps.